CMakeLists.txt

```diff
@@ -361,6 +350,14 @@ endif(BUILD_HELICS_EXAMPLES)
 INSTALL(FILES ${KEY_LIBRARY_FILES}
 	DESTINATION ${CMAKE_INSTALL_BINDIR}
 	COMPONENT runtime)
+
+install(DIRECTORY ${AUTOBUILD_INSTALL_PATH}/lib/ DESTINATION ${CMAKE_INSTALL_LIBDIR}/ OPTIONAL)
+install(DIRECTORY ${AUTOBUILD_INSTALL_PATH}/lib64/ DESTINATION ${CMAKE_INSTALL_LIBDIR}/ OPTIONAL)
+
+if (HELICS_HAVE_ZEROMQ)
+	install(FILES $<TARGET_LINKER_FILE:libzmq> DESTINATION ${CMAKE_INSTALL_LIBDIR} COMPONENT libraries)
```

Comment:
This causes problems when a system copy of ZeroMQ is used. On LC this copied the libzmq.so symlink, which points to libzmq.so.5.0.1; I think for system copies of ZeroMQ, we probably don't want to copy it, or if we do we need to copy the file pointed to by the symlink (which could be another symlink).

Reply:
I think I meant that to be restricted to windows and not msys.